src/Plugin/search_api/processor/Property/EntityReferenceWithUriProperty.php~

```diff
@@ -0,0 +1,62 @@
+<?php
```

Comment:
Swap file?

Reply:
oh no! new vim configs. Thanks for catching!